portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,82 @@
+function loadInFact(index, fact_index, isTruth) {
+    let fact = isTruth ? truths[fact_index] : lies[fact_index];
+
+    switch(index) {
+        case 0:
+            document.getElementById("first_txt").innerHTML = fact[0];
+            document.getElementById("first_img").src = fact[1];
+            break;
+        case 1:
+            document.getElementById("second_txt").innerHTML = fact[0];
+            document.getElementById("second_img").src = fact[1];
+            break;
+        case 2:
+            document.getElementById("third_txt").innerHTML = fact[0];
+            document.getElementById("third_img").src = fact[1];
+            break;
+    }
+}
+
+function loadInTruth() {
+
+    let first_truth_value = Math.floor(Math.random() * truths.length);
+    let second_truth_value = Math.floor(Math.random() * truths.length);
+    while (first_truth_value == second_truth_value) {
+        second_truth_value = Math.floor(Math.random() * truths.length);
+    }
+
+    let first_truth_index = Math.floor(Math.random() * 3);
+    let second_truth_index = Math.floor(Math.random() * 3);
+    while (first_truth_index == second_truth_index) {
+        second_truth_index = Math.floor(Math.random() * 3);
+    }
+
+    loadInFact(first_truth_index, first_truth_value, true);
+    loadInFact(second_truth_index, second_truth_value, true);
+
+    return [first_truth_index, second_truth_index];
+}
+
+
+function loadInLie(truth_indeces) {
+    //determine the location without a truth value
+    let lie_index = (3 - (truth_indeces[0] + truth_indeces[1]));
+    
+    let lie_value = Math.floor(Math.random() * lies.length);
+    loadInFact(lie_index, lie_value, false);
+
+    return lie_index;
+}
+
+function next(answer)  {
+    let q_num = document.getElementById("question_num");
```

Comment:
Considering the context of my comments on abbreviations and the naming of this element above, I'd call this variable something like gameHeading or gameHeadingElement or something to indicate that it represents the area of the page that could contain either a prompt to start the game or a question.

Also, since you never reassign this variable to equal anything else, I think you should use const instead of let.

const gameHeading = document.getElementById('game-heading');

Or something like that.

Reply:
Changed